org.scantailor.ScanTailor.json

```diff
@@ -0,0 +1,61 @@
+{
+  "app-id": "org.scantailor.ScanTailor",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "scantailor",
+  "rename-appdata-file": "scantailor.appdata.xml",
+  "rename-desktop-file": "scantailor.desktop",
+  "rename-icon": "scantailor",
+  "finish-args": [
+    "--socket=x11",
+    "--share=ipc",
+    "--filesystem=home"
+  ],
+  "modules": [
+    "shared-modules/qt4/qt4-4.8.7-minimal.json",
+    {
+      "name": "boost",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://dl.bintray.com/boostorg/release/1.68.0/source/boost_1_68_0.tar.bz2",
+          "sha256": "7f6130bc3cf65f56a618888ce9d5ea704fa10b462be126ad053e80e553d6d8b7"
+        }
+      ],
+      "buildsystem": "simple",
+      "build-commands": [
+        "./bootstrap.sh --prefix=/app",
+        "./b2 install"
+      ],
+      "cleanup": [
+        "/include",
+        "/lib/*.a"
+      ]
+    },
+    {
+      "name": "scantailor",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/scantailor/scantailor/archive/RELEASE_0_9_12_1.tar.gz",
+          "sha256": "ef5d5bdca207ab00701121a32e9b95c7c7353c642b9538b3f9ca040d8d1a5dde"
+        },
+        {
+          "type": "file",
+          "path": "scantailor.desktop"
+        },
+        {
+          "type": "file",
+          "path": "scantailor.appdata.xml"
+        }
+      ],
+      "buildsystem": "cmake-ninja",
+      "post-install": [
+        "install -Dm0644 resources/appicon.svg /app/share/icons/hicolor/scalable/apps/scantailor.svg",
```

Comment:
You could just install those files with the correct name and remove the rename-appdata-file & rename-desktop-file & rename-icon.

Reply:
Not quite, because those property change values inside to be correct.